### DISCUSSION TITLE
英文摘要中的英文长标题可以使用什么方式换行

### DISCUSSION THREAD

**Discussion by Starrynight1234** [nju-lug/NJUThesis#105] [Q&A]
使用“//”好像并不能起到效果，现在的版本自动换行会把某个单词拆分开来，不太美观

**Comment by atxy-blip** [comment]
与 <https://github.com/nju-lug/NJUThesis/issues/98> 重复。事实上已经差不多写好了，用的是 TeX 原生的断词算法，效果很自然。还有些边界条件要完善一下，也许周末能推上去。

我没有想着给摘要标题开放手动换行的功能，因为会和用在封面的标题冲突，总不能指定两种手动换行方式吧。

![image](https://user-images.githubusercontent.com/63113546/159500363-0bae8a45-3fd9-49ed-aba3-6d1a9e723d26.png)

-----

https://github.com/nju-lug/NJUThesis/commit/15ad993c69f5c9e4cf8cfb39845a721a8aa016fe 已修复。

**Comment by Starrynight1234** [comment]
好的，谢谢！
